client/me/purchases/manage-purchase/auto-renew-toggle/index.jsx

```diff
@@ -139,16 +143,31 @@ class AutoRenewToggle extends Component {
 		return this.props.isEnabled;
 	}
 
+	renderTextStatus() {
+		const { translate, isEnabled, withPlan } = this.props;
+
+		if ( withPlan ) {
+			return isEnabled ? translate( 'Plan auto-renew (on)' ) : translate( 'Plan auto-renew (off)' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Auto-renew is OFF' )` *ES Score: 6*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-status-autorenew-toggle)